<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerWidget</class>
 <widget class="QWidget" name="PlayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#playPauseButton,
#prevButton,
#nextButton {
	color: #fff;
	border-style: solid;
	border-width:0px;
	background-color: rgba(255,72,0,.8);
}

#playPauseButton:hover,
#prevButton:hover,
#nextButton:hover {
	background-color: rgba(255,72,0,.7);
}

#playPauseButton:pressed,
#prevButton:pressed,
#nextButton:pressed {
	background-color: rgba(255,72,0,.6);
}</string>
  </property>
  <widget class="QPushButton" name="nextButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>65</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius:20px;
}</string>
   </property>
   <property name="text">
    <string> ▶▶|</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>65</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius:20px;
}</string>
   </property>
   <property name="text">
    <string>|◀◀ </string>
   </property>
  </widget>
  <widget class="QSlider" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>281</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QSlider::groove:horizontal {
     border: 1px solid #999999;
     height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
     margin: 2px 0;
 }

 QSlider::handle:horizontal {
     background: rgba(255,72,0,.8);
     border: 1px solid #5c5c5c;
     width: 18px;
     margin: -2px 4px; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
     border-radius: 3px;
 }

QSlider::sub-page:horizontal {
	background: rgba(255,72,0,.8);
	margin: 3px 10px 3px 1px;
}
</string>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="tracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="artworkLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/icons/placeholder.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="creatorNameLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>45</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>creatorNameLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="soundNameLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>soundNameLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="playPauseButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>60</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border-radius: 25px;
}</string>
   </property>
   <property name="text">
    <string> ▶</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
